components/prism-elixir.js

```diff
@@ -1,4 +1,7 @@
 Prism.languages.elixir = {
+	'doc': {
+		pattern: /@(?:doc|moduledoc)\s+("{1,3}|'{1,3})(?:\\[\s\S]|(?!\1)[^\\])+\1/
```

Comment:
`"{1,3}|'{1,3}` is not a good idea.

```suggestion
		pattern: /@(?:doc|moduledoc)\s+(?:("|')(?:\\(?:\r\n|[\s\S])|(?!\1)[^\\\r\n])*\1|("""|''')[\s\S]*?\2)/
```

Reply:
This pattern breaks multiline strings
![](https://p2.f0.n0.cdn.getcloudapp.com/items/yAu6n7p6/4ba1edac-1a9e-4388-8536-98e152f31983.png?source=viewer&v=e52509da8d1dff5862cc975b5cdef314)

I've pushed one that works - by my regex foo isn't great - so feel free to suggest a better one.

Reply:
> This pattern breaks multiline strings

My bad. I messed up the order. I'll suggest a pattern that works...